sdk/src/main/scala/com/horizen/storage/SidechainStateStorage.scala

```diff
@@ -77,48 +81,129 @@ class SidechainStateStorage (storage : Storage, sidechainBoxesCompanion: Sidecha
     }
   }
 
-  def update(version : ByteArrayWrapper, withdrawalEpochInfo: WithdrawalEpochInfo,
-             boxUpdateList : Set[SidechainTypes#SCB],
-             boxIdsRemoveList : Set[Array[Byte]],
-             withdrawalRequestAppendList : Set[WithdrawalRequestBox]) : Try[SidechainStateStorage] = Try {
+  def getConsensusEpoch: Option[ConsensusEpochNumber] = {
+    storage.get(consensusEpochKey) match {
+      case v if v.isPresent =>
+        Try {
+          Ints.fromByteArray(v.get().data)
+        } match {
+          case Success(epoch) => Some(intToConsensusEpochNumber(epoch))
+          case Failure(exception) =>
+            log.error("Error while consensus epoch information parsing.", exception)
+            Option.empty
+        }
+      case _ => Option.empty
+    }
+  }
+
+  def getForgingStakesAmount: Option[Long] = {
+    storage.get(forgingStakesAmountKey) match {
+      case v if v.isPresent =>
+        Try {
+          Longs.fromByteArray(v.get().data)
+        } match {
+          case Success(epoch) => Some(epoch)
+          case Failure(exception) =>
+            log.error("Error while forging stakes amount parsing.", exception)
+            Option.empty
+        }
+      case _ => Option.empty
+    }
+  }
+
+  def getForgingStakesInfo: Option[Seq[ForgingStakeInfo]] = {
+    storage.get(forgingStakesInfoKey) match {
+      case v if v.isPresent =>
+        forgingStakeInfoSerializer.parseBytesTry(v.get().data) match {
+          case Success(stakesInfo) => Some(stakesInfo.asScala)
+          case Failure(exception) =>
+            log.error("Error while forging stakes parsing.", exception)
+            Option.empty
+        }
+      case _ => Option.empty
+    }
+  }
+
+  def update(version: ByteArrayWrapper,
```

Comment:
shall we make parameters for an update as just case class? Btw case class to list for updating of storage could be described in such case class as well